lib/selection-count-view.coffee

```diff
@@ -22,8 +35,12 @@ class SelectionCountView extends HTMLElement
 
   updateCount: ->
     count = @getActiveTextEditor()?.getSelectedText().length
+    lineCount = @getActiveTextEditor()?.getSelectedText().split(/\r\n|\r|\n/).length
```

Comment:
Instead of splitting here which I imagine might be a big sluggish on really large selections, you can access the selection range directly via:

```coffee
lineCount = @getActiveTextEditor().getSelectedBufferRange().getRowCount()
```